### DISCUSSION TITLE
Timing on support for OpenSSH 9.6 ?

### DISCUSSION THREAD

**Discussion by dave-re-imprivata** [rapier1/openssh-portable#59] [General]
With the advent of Terrapin and the keystroke timing obfuscation vulnerability, we're going to skip 9.5... any idea when support for 9.6 might be available? Thanks!

**Comment by rapier1** [comment]
Hey there,

Sorry of the delay I've been on break until today. We've started the process of porting to OpenSSH 9.6. The initial tests are looking good and we hope to have something out in the next few days. The DynamicWindow only release may be available earlier as there is as much testing required. 

Chris

**Comment by dave-re-imprivata** [comment]
Thanks, Chris - no need to cut Dynamic Window earlier on our end. We'll pick it up when you do the full release. Thanks!
